exporter/api.py

```diff
@@ -37,8 +38,30 @@ def sanitize(title):
                 id=learning_resource.id,
                 title=sanitize(learning_resource.title),
             )
-            with open(os.path.join(tempdir, filename), 'w') as f:
+            type_name = learning_resource.learning_resource_type.name
+            try:
+                os.mkdir(os.path.join(tempdir, type_name))
+            except OSError:
+                pass
```

Comment:
It prevents against the race condition where you check if the directory exists, it does not at the time, then another process creates the directory and then you get an exception during `mkdir` anyway.